account_invoice_export_server_env/models/transmit_method.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class TransmitMethod(models.Model):
+    _name = "transmit.method"
+    _inherit = ["transmit.method", "server.env.mixin"]
```

Comment:
I'd use https://github.com/OCA/server-env/blob/13.0/server_environment/models/server_env_tech_name_mixin.py as messing up conf by changing display names is quite easy ;)

Reply:
As the `transmit_method` model as already a field named `code` that act as a technical field and is unique.
I did a change to use that one (see line below)